aspnet/hosting/index.rst

```diff
@@ -0,0 +1,15 @@
+Hosting
+=======
+
+By `Sourabh Shirhatti`
+
+The hosting docs provide an overview of the features on ASP.NET 5. Hosters who offer ASP.NET hosting will be able to use the setup and configuration recommendations to integrate ASP.NET 5 into their hosting solution. This document should also serve as a guide for any enviroment where multiple discrete users will be running web sites on a single server or web farm.
```

Comment:
No they don't. "provide an overview of how to host ASP.NET 5", not "of the features on"